.github/CODEOWNERS

```diff
@@ -2,22 +2,22 @@
 
 # These owners will be the default owners for everything in
 # the repo. Unless a later match takes precedence.
-@mackec1 @keithdadkins @lsmitchell @mjburling @brianburton 
+@mackec1 @keithdadkins @lsmitchell @mjburling @brianburton @brandoncruz3
```

Comment:
Is it normal/new that it complains about the user not existing here? (Or is that just since hes not added yet?)